transactions/reset-epoch/2023/oct-18/arguments.json

```
@@ -0,0 +1 @@
+[ 	{ 		"type":"UInt64", 		"value":"??" 	}, 	{ 		"type":"String", 		"value":"f28339976b22b61917e69fb5eca9eeb8" 	}, 	{ 		"type":"UInt64", 		"value":"7000000" 	}, 	{ 		"type":"UInt64", 		"value":"200000000" 	}, 	{ 		"type":"UInt64", 		"value":"300000000" 	} ]
```

```suggestion
[ 	{ 		"type":"UInt64", 		"value":"97" 	}, 	{ 		"type":"String", 		"value":"f28339976b22b61917e69fb5eca9eeb8" 	}, 	{ 		"type":"UInt64", 		"value":"8000000" 	}, 	{ 		"type":"UInt64", 		"value":"200000000" 	}, 	{ 		"type":"UInt64", 		"value":"300000000" 	} ]
```
Also, next time, lets make these as separate lines so it is easier to leave a comment